.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish to NuGet
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
@xperiandri same point I mentioned about filtering to master branch applies here